apps/right-to-rent-check/fields/index.js

```diff
@@ -31,5 +31,21 @@ module.exports = {
   },
   'tenant-dob': dateComponent('tenant-dob', {
     validate: ['required', 'date', 'before']
-  })
+  }),
+  'agent-company': {
+    mixin: 'input-text',
+    validate: 'required'
+  },
+  'agent-name': {
+    mixin: 'input-text',
+    validate: 'required'
+  },
+  'agent-email-address': {
+    mixin: 'input-text',
+    validate: ['required', 'email']
+  },
+  'agent-phone-number': {
+    mixin: 'input-text',
+    validate: ['required']
```

Comment:
just a bit of inconsistency here with line 41 where `validate` is not in square braces.